### ISSUE TITLE
Success Criterion 1.4.4 - Resize Text - Level AA

### ISSUE THREAD

**Issue by JJdeGroot** [w3c/matf#3]
Discussions:

<details>
<summary>15 April 2024</summary>

[Source](https://www.w3.org/2024/04/15-matf-minutes.html#t04)

## 1.4.4 Resize text

JJ: question: does resize text apply?

JJ: what's interesting is that OS settings can be used for this… with websites, this works differently

JJ: with apps it is trickier to determine the percentage…and in iOS there is dynamic font sizing now

JJ: that makes it even harder to determine how much textt even has scaled. And we cannot inspect tex

QuintinB: what's completely missing… not sure how it works in iPhone but in Android, because we have scale independent sizes, we could set a minimum font size. This has been missing from WCAG since forever

QuintinB: so you could say this is the smallest we allow and then go to scaling

<Joe_Humbert> on iOS they have typography in the human interface guidelines that give the pt size for all font styles and font sizes

JJ: wonder if we can add some kind of recommendation in guidelines

Carolina: if we're able to scale / zoom in and out, would only one be required or both of them?

Carolina: on iOS I use large, on Android I use the maximum

Carolina: do we need to allow for both?

joe_humbert: do you mean in hybrid situations?

Carolina: I mean if you have the web view should you be able to pinch zoom and out?

<QuintinB> The Evinced MCAG reduces everything to pixel sizes, maybe that is a logical way to go

<QuintinB> I do think we need to have a "standard minimum" font size

JJ: another issue is that WCAG uses CSS pixels wjhich for apps you'd ned to convert

<Jamers> I would strongly suggest to drop the "200%" spec from a mobile interpretation; 200% or 2x is very web-based thinking particularly for larger heading font styles

Karen: in the passed we said we should not disable scaling

Karen: but others say now that as long as magnification works that suffices, have you discussed that here?

<QuintinB> I don't think relying on magnification because that may be seriously inconvenient for many people

JJ: don't think we have discussed that here

JJ: is it sufficient if an app has its own resizing mechanism?

<julianmka> If the device supports actual text scaling through setting, apps should support that capability.

<QuintinB> agree with julianmka

<QuintinB> But at what point does it need to support the font size without loss of content or functionality?

JJ: the bold font setting is also interesting, lots of users (10%) use it we found in research

<QuintinB> woah - can we get references for that?

<JJ> Bold text stats: https://appt.org/en/stats/bold-text

<QuintinB> I love me some sweet sweet data

<QuintinB> thanks

<julianmka> "But at what point does it need to support the font size without loss of content or functionality?" -- show it all, let users determine whether it's functional/meets their needs

<Mick> Also agree with julianmka

<Jamers> +1 julianmka

<Jamers> making it a part of the SC

JJ: in iOS / Android there is a limit too

<QuintinB> It would be interesting to know how the web folks arrived at 200%

JJ: not sure if there is a limit on websites

<Jamers> +! QuintinB

<QuintinB> Android can be done via ADB, it's just a multiplier as a float value

JJ: there are all users who decrease font sizes

Julian: could we include some kind of statement that when an OS supports a specific AT or accessibility setting, that app developers should support it?

<JJ> Mobile Device Features sheet: https://docs.google.com/spreadsheets/d/1j1-C7t4sHtJqLT4YKJqLfJIl28DET6gYBQPczDFDWpY/edit#gid=0

Julian: we often hear about users with edge cases…

Joe_Humbert: should be careful about 'should' vs 'must'… eg there could be features that are newly released and developers might not have time to explore new features before they are released, could take months

Julian: valid point

<QuintinB> On the other hand, you get settings that don’t get respected for months, to the point where the documentation actually says that it’s not usually respected because developers don’t know about it (e.g. android time to react)

JJ: maybe we can fort certain features, like text scaling, but not others

<QuintinB> Sorry, years, not months

<Mick> Could state support must be given for a certain OS version, so not to put pressure around new releases

Jamie: are you sharing the doc because you feel it should be updated?

JJ: would be good to keep an eye on new features and discuss in these meetings how we deal with the new features

JJ: in the European rules there are a lot of requirements around user preferences, like color preferences

<Joe_Humbert> I think with Assistive technology/settings we could recommend a support timeline. Like support new AT within 1-2 years, but not sure that follows other specs/notes

hdv: is it EN 301 549?

<Jamers> I don't see Full Keyboard Access on the linked document, which is available since 1OS 14. Did I miss it?

JJ: yes, 3.2.1 of that

JJ: feel free to add any features that are missing
</details>

**Comment by jamieherrera** [comment]
1.4.4 reference to "without assistive technology" would be helpful to clarify in the documentation whether text sizing in an Accessibility section such as mobile Chrome browser, would be "assistive technology" , whether for mobile web or mobile apps.

**Comment by jamieherrera** [comment]
Would mobile web be exempt from this SC? 
Some of my work colleagues feel strongly that 1.4.10 for testing desktop web is sufficient to catch any content or functionality issues on small screens, and that the informative ACT rule on the viewport meta tag https://www.w3.org/WAI/standards-guidelines/act/rules/b4f0c3/ enables pinch to zoom by not disabling the viewport meta tag. Since type of text resize / zoom is not defined, the argument is that the ability to pinch to zoom is a sufficient test for this SC for mobile web.

**Comment by JJdeGroot** [comment]
<details>
<summary>Discussion during MATF meeting on June 12, 2024</summary>

[Source](https://www.w3.org/2024/06/12-matf-minutes.html#t01)

Jamie: elaborating on comment to Github issue on 1.4.4

(audio problems)

victoria: it should be a violation if there is an option to increase text size in the OS
… agree that pinch to zoom satisfies 1.4.4 but it may not wor for all...

<JJ> Detlev: explaining German ways of testing 1.4.4

<AlainVagner> EN 301 549 criterion 11.7 "User preferences": https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf#page=82

Detlev explains line of though linking 1.4.4 ans 11.7 (EN 301 549)
</details>

**Comment by jamieherrera** [comment]
note updated text in WCAG2ICT draft  https://w3c.github.io/wcag2ict/#resize-text - should we be discussing/ including terms like Closed Functionality from WCAG2ICT in our Mobile supporting doc?

**Comment by JJdeGroot** [comment]
Related discussion in WCAG2ICT: https://github.com/w3c/wcag2ict/issues/4

It's challenging to understand the difference between assistive technologies and accessibility features.
The usage of assistive technologies would fail ("without needing to use assistive technologies.") while accessibility features would pass ("the application works with the platform accessibility features").

[Assistive technologies](https://w3c.github.io/wcag2ict/#dfn-assistive-technologies) is defined as term and lists "screen magnifiers, and other visual reading assistants" in the examples.

So it seems that zoom functionality provided on mobile operating systems would fall in the assistive technology category and not in the platform accessibility features category.

The "three tap zoom" functionality is not mentioned explicitly, but considering that's also specifically targeting visual impairments it would probably as be categorised as assistive technology.

I think it would really help app developers if we would clearly distinguish which functionality falls in the assistive technology category, and which falls in the accessibility features category.

**Comment by doesDesign** [comment]
Thank you @JJdeGroot for bumping this one! I created an issue at [w3c/wcag2ict/issues/527](https://github.com/w3c/wcag2ict/issues/527) because I wasn't able to tag a few W3C folks I had in mind. I agree it's very challenging especially with this new guidance in WCAG2ICT

**Comment by JJdeGroot** [comment]
Based on info from https://github.com/w3c/wcag2ict/issues/527 and https://github.com/w3c/matf/issues/67#issuecomment-2489115618, it seems that:

- Following the normative text of WCAG2ICT, system zoom would be a way to pass 1.4.4, even if text does not scale.
- Following the normative of EN 301 549, you would not pass because you would not honor the user preferences for font size defined in Chapter 11.7

**Comment by JJdeGroot** [comment]
Some research about non-linear text scaling:

## iOS

Starting from iOS 11, text is scaled using [Dynamic Type](https://developer.apple.com/documentation/uikit/uifont/scaling_fonts_automatically/). The factors are defined in [Typography Specifications](https://developer.apple.com/design/human-interface-guidelines/typography#Specifications).

The default scale is `Large`

| Style       | Weight   | Size (points) | Leading (points) |
|-------------|----------|---------------|------------------|
| Large Title | Regular  | 34            | 41               |
| Title 1     | Regular  | 28            | 34               |
| Title 2     | Regular  | 22            | 28               |
| Title 3     | Regular  | 20            | 25               |
| Headline    | Semibold | 17            | 22               |
| Body        | Regular  | 17            | 22               |
| Callout     | Regular  | 16            | 21               |
| Subhead     | Regular  | 15            | 20               |
| Footnote    | Regular  | 13            | 18               |
| Caption 1   | Regular  | 12            | 16               |
| Caption 2   | Regular  | 11            | 13               |

For body text, the default is 17 points, meaning we need to reach 34 points for 200% scaling.

At `AX2` the size is 33 points, meaning we need to choose `AX3` which is 40 points (235% scaling)

I've calculated the percentages for each size from `Large` to `AX3`:

| Style       | Weight   | Size (points) | Leading (points) |
|-------------|----------|---------------|------------------|
| Large Title | Regular  | 52 (153%)           | 61               |
| Title 1     | Regular  | 48 (171%)            | 57               |
| Title 2     | Regular  | 44 (200%)           | 52               |
| Title 3     | Regular  | 43 (215%)            | 51               |
| Headline    | Semibold | 40 (235%)            | 48               |
| Body        | Regular  | 40 (235%)            | 48               |
| Callout     | Regular  | 38 (238%)            | 46               |
| Subhead     | Regular  | 36 (240%)            | 43               |
| Footnote    | Regular  | 33 (254%)            | 40               |
| Caption 1   | Regular  | 32 (267%)            | 39               |
| Caption 2   | Regular  | 29 (264%)            | 35               |

## Android

Starting from Android 14, [text is scaled non-linearly](https://developer.android.com/about/versions/14/features#non-linear-font-scaling). The docs tell you how to convert values, but there is no table indicating which scaling factor is applied.

Luckily, Android is open-source, the [FontScaleConverterFactory.java](https://android.googlesource.com/platform/frameworks/base/+/refs/heads/main/core/java/android/content/res/FontScaleConverterFactory.java) class holds the non-linear scaling values. The values are generated using [font-scaling-array-generator.js](https://android.googlesource.com/platform/frameworks/base/+/android-14.0.0_r1/tools/fonts/font-scaling-array-generator.js)

The definition:

```java
/* fromSp= */
new float[] {   8f,   10f,   12f,   14f,   18f,   20f,   24f,   30f,  100},
/* toDp=   */
new float[] {  16f,   20f,   24f,   26f,   30f,   34f,   36f,   38f,  100})
```

In table form:

| Base Size | Scaled Size | Scale |
|-----------|-------------|------|
| 8 | 16 |  200% | 
| 10 |  20 |  200% | 
| 12 |  24 |  200% | 
| 14 |  26 |  186% | 
| 18 |  30 |  167% | 
| 20 |  34 |  170% | 
| 24 |  36 |  150% | 
| 30 |  38 |  127% | 
| 100 |  100 |  100% | 

It seems that text larger than 14 sp will never reach 200% text scaling on stock Android.

**Comment by JJdeGroot** [comment]
Discussed in today’s meeting.

<details>
<summary>29 January 2025</summary>

[Source](https://www.w3.org/2025/01/29-matf-minutes.html)

~~~text
JJ: Reads out WCAG text for 1.4.4

JJ: Reads out WCAG2ICT and note

JJ: note refers to platform options for text scaling (Zoom)

<Jamie> can we add "nonlinear scaling" to the terms list?

JJ: reads out note that some text may not scale fully to 200%

JJ: Side effect of replacement user agent > platform is that in native context the OS zoom function would be sufficient

JJ: Discussion in WCAG2ICT github, result is that OS zoom would pass
… Android scaling is not linear (large text scales less)

<quintinb> Does this mean that you can fix font sizes and as long as the magnifier service magnifies, that's acceptable?

<JJ> [w3c/matf#3](https://github.com/w3c/matf/issues/3)

JJ: So one of the issue is what can be exempted in native apps when text scaling

JJ: We may need a note on large content viewer (iOS) as method tzo comply

<JJ> close the queue

Jamie: One issue in the wild is exceptions for some copmponents in mobile apps, some component types don't scale - we should cature what components can scale and what othe rcomponents can't

JJ: Yes we should point out thosde limitations

<Jon_Gibbins> +1 Jamie, although we need to be careful that behaviours may change rapidly, so should form part of non-normative text (e.g. understanding)

Joe_Humbert: We should be careful - the large content viewe in iOS has no equivalent in Android - so you may need an equivalent in Android

<Jon_Gibbins> Or perhaps such differences in support or behaviours can be in SC notes?

<Joe_Humbert> thank you Detlev for scribing

<quintinb> Thanks all, and especially Detlev for scribing!

WOnder it scaling issues would ever FAIL anything if OS zoom is considered sufficient...

<JJ> We might also need more emphasis on the scenarios mentioned in Understanding doc

<JJ> "Some user interface components that function as a label and require activation by the user to access content are not wide enough to accommodate the label's content. For example, in Web mail applications the subject column may not be wide enough to accommodate every possible subject header, but activating the subject header takes the user to the full message with the full subject header." https://www.w3.org/WAI/WCAG22/Understanding/resize-text.html
~~~
</details>

The discussion will be continued next week.

**Comment by detlevhfischer** [comment]
Is there any way at all to support users' text scaling settings in webviews (iOS, Android)? 

If it is not supported, would apps using webviews immediately fail 1.4.4 if we should decide that Zoom must be considered an assistive technology and not sufficient to meet 1.4.4?

**Comment by JJdeGroot** [comment]
Discussed in today’s meeting.

<details>
<summary>5 February 2025</summary>

[Source](https://www.w3.org/2025/02/05-matf-minutes.html#53b8)

~~~text
Also Android lets you define really awful text spacing

1.4.4. should not require the use of a specific assistive technology

Detlev Are we all in agreement that zoom alone is sufficient to pass. It's not best practice, I would not agree that it's not clear. Be careful with that statement

I think we just need to be careful that we don't get prescriptive about specific AT

JJ reads 1.4.4 "Except for captions and images of text, text can be resized without assistive technology up to 200 percent without loss of content or functionality. " - note that it's without the use of AT

Jon_Gibbins My understanding has always been that it's without AT. The reading has been changed to actually be explicit in not requiring a magnifier. It does mention this in the understanding doc

Jon_Gibbins I think we should continue that in the mobile guidelines. We need to use WCAG as the original intent

Tanya it feels to me (the definition of zoom) - it means something different. In web zoom intends to do something different than a magnifier. It seems that screen zoom and magnifying from a function perspective is a bit confusing. The definition needs to be more broad.

Jon_Gibbins these guidelines are for the builders and the responsibilities on them. Zoom works indepently of the development. Font size is within the control of developers. A solutions perspective is fine, but these guidelines are aimed at buildersa

<JJ> WCAG2ICT comment: [w3c/wcag2ict#4 (comment)](https://github.com/w3c/wcag2ict/issues/4#issuecomment-2183357940)

<JJ> "Point 3: We've long held that PC magnifier programs are assistive technology and therefore not a method of meeting 1.4.4. Is the same true on other platforms?

<JJ> TF Answer to Point 3: For Non-Web Documents and Software, features including software provided by the platform that provide a means of enlarging the text 200% (zoom or otherwise) without the loss of content or functionality, meet the intent of this success criteria.

<JJ> Platform accessibility features, including platform software that, when applied, causes loss of content, including a reduction in the ability to distinguish characters, would not meet this success criteria."

Joe_Humbert We might need to bring this back to WCAG2ICT because the linked issue has a conflicting statement in the note ([w3c/wcag2ict#4 (comment)](https://github.com/w3c/wcag2ict/issues/4#issuecomment-2183357940)) there is an answer to point 3... it does seem to conflict the draft. The draft says in note 1 that the application provides some means... I'm confused because

they answered the question then changed it in the note

<JJ> Versions of WCAG2ICT: https://www.w3.org/standards/history/wcag2ict-22/

Detlev A comment on Jon_Gibbins - I think looking at WCAG there are many cases where something should be possible for the user. There are different options available. Sometimes the author doesn't need to do anything in order to pass. It's not uncommon that the author doesn't need to do anything to ensure things are available to users, which may be

the case in terms of font scaling

<Jon_Gibbins> “Zoom” as a feature is a bit ambiguous – and I’m uncertain that WCA2ICT discussion is referring to Magnification AT. “Zoom” on a web browser is different to, and behaves differently to “zoom” accessibility features of a mobile OS.

JJ let's continue the discussion on GitHub

<Detlev> if you requre text scaling you end up with many situations whee not all text scales - and where do you draw the line then if something passes or fails?

<Jon_Gibbins> I’d just be worried about suggesting to mobile app developers that it’s okay to not support dynamic type because users can just use magnification. (Noting that we are focussed on mobile, and for broader ICT the story may be different – kiosks for example.)

jamie: Do we need something here on custom components and system font sizes to account for authors using custom font sizes.

<julianmka> +1 Jon

Joe_Humbert do we need a vote / discussion on where Zoom fits in?

Jon_Gibbins if Regex is your solution, you have 2 problems

ACTION: Create issue for terminology used in Android and iOS for Zoom related functions

<Jon_Gibbins> quintinb: Haha - habit. Sorry.

:D

Jamie: Did we talk about hybrid content and how to interpret zoom for hybrid content. for e.g. reactive native let's you use system sizing or other platformsa that allow pinch to zoom
~~~
</details>

- ACTION: Create issue for terminology used in Android and iOS for Zoom related functions

**Comment by jamieherrera** [comment]
An action item for today's discussion discussed better defining "assistive technology". 

As a general action item that is fine as well, but for [WCAG 1.4.4](https://www.w3.org/TR/WCAG22/#resize-text) it would only need to be scoped to text resizing: 

> "text can be resized without assistive technology"

Other than Zoom as an Accessibility feature, are we essentially determining if the resize text features that are only found in a device's accessibility-specific section considered assistive technology?

Or are we just asking if Zoom is an assistive technology?

**Comment by JJdeGroot** [comment]
Discussed in meeting on 14 May 2025.

<details>
<summary>20 May 2025</summary>

[Source](https://www.w3.org/2025/05/14-matf-minutes.html#b871)

~~~text
Joe_Humbert: Very large topic

Joe_Humbert: (reading WCAG text, WCAG2ICT text)
… "without use of AT"

Joe_Humbert: Discussions were about viewport size, device specific text size settings and scaling behaviour
… supporting OS specific AT settings

Joe_Humbert: note on timeline for supporting features
… discussion on whether mobile is exempt from 1.4.4 (pending on the meaning of AT)
… requesting clarificatino on defining what counts as AT

<rachaely> +1 to defining Assistive Technologies

<Joe_Humbert> [w3c/wcag#730 (comment)](https://github.com/w3c/wcag/issues/730#issuecomment-2878110621)

Joe_Humbert: JJ commented based on issue response that on native mobile, system zoom would be a way to pass 1.4.4
… also added info on sizes styles available on native mobile
… there was discussion on large content viewer in iOS
… concern on too much detail on techniques since tech changes quickly

<Joe_Humbert> [w3c/matf#3](https://github.com/w3c/matf/issues/3)

Joe_Humbert: mostly about the line what is AT, what isn't?

<Joe_Humbert> close the queue

julianmka: We can lean on the fact that text resizing has robust support in OSs so that is what people need to do

Joe_Humbert: Greggs distinction still does not give full clarity regarding what is AT

Joe_Humbert: More clarity on product segments on desktop, less for mobile

<Jamie> congrats to Joe_Humbert for leading

<Jamie> er thank you

<Joe_Humbert> Continue discussing SC 1.4.4 Next meeting
~~~
</details>

**Comment by JJdeGroot** [comment]
Discussed in meeting on 21 May 2025.

<details>
<summary>21 May 2025</summary>

[Source](https://www.w3.org/2025/05/21-matf-minutes.html#314f)

~~~text
<pauljadam> Screen Magnifier is an Assistive Technology. Text Size setting is not an AT.

JJ summarizes what has been discussed last week. Definitions, system zoom, definition of AT as a key of what can be used to pass.

<pauljadam> Apple considers Larger Accessibility Text Sizes as an "accessibility setting" NOT an Assistive Technology.

<pauljadam> Do we need a note to encourage authors to enable text scaling up to the maximum possible size on the user's device?

JJ JJ reads normative text, available definitions and notes; points out the difference between mainstream users and users of AT.

<Jamie> @Paul I'd say no

<pauljadam> Or some note about how it's hard to know when you are at 200%?

<GleidsonRamos> +1 about the note on how hard it is to know what is 200%. Dynamic scaling make that's harder

<pauljadam> I agree about the Large Content Viewer we may want to mention that also

pauljadam Do we need a note how its hard to know whether the text is scaled to 200%

<JJ> Tanya can you add ":" after the name of speaker, will look nicer in minutes :) e.g. JJ: blabla

RobW: you don't always get to 200%, so you would argue if it would ever pass. Users care that it works.

<RobW> you could scale text in bars, but that would be the wrong thing to do.

Jamie: asks whether we need to consider and how to approach the topic of customization of the fonts

<pauljadam> Pinch to zoom would only be allowed on a web view inside a native app.

<pauljadam> There is also a way to make web views respect the user's text size preferences with special apple CSS.

<pauljadam> Some apps will disable pinch to zoom in their web views which would be an easy fail if they're not supporting the user's text size preferences.

JJ: this SC was intended for web where you have zoom function and everything will scale without truncation. On iOS and Android it's working differently. Pinch-to-zoom will not be allowed on mobile. Web makes some things a lot easier.

<pauljadam> Screen Magnifier would never be allowed to pass this SC.

<JJ> https://w3c.github.io/matf/#success-criterion-1-4-4-resize-text

<JJ> https://www.w3.org/WAI/WCAG22/Understanding/resize-text.html

<pauljadam> Zoom is not ok

Illai: intend of this SC is not limited to zoom or other techniek. Zoom is okey.

<pauljadam> WCAG defines screen magnifier as an AT

<RobW> I can't agree that a screen magnifier can be used to pass this SC

JJ: we need to have a definition of AT

<Illai> pauljadam the reference is to the user agent Zoom (control +, or command +)]

pauljadam: resize text is accessibility technology. Developes can ignore this all and not nothing at all

JJ: display scaling can be a way to pass. I feel like magnifier is an AT

Jamie: question of accessibility vs usability

<pauljadam> I don't think the iOS Display & Brightness > Display Zoom setting would be considered an Assistive Technology either.

<RobW> Agreed with PJA

<Jamie> @Tanya specifically "loss of content or functionality" identifying accessibility / 1.4.4 failure vs non-WCAG UX design

<pauljadam> Yes agree that the user's text size preferences should be respected!

<JJ> Tanya: do we agree as group that user preferences should be respected?

<Jamie> user preference = OS settings, not specific %

<pauljadam> That's what the user's expect. All sizes should work.

JJ: Apple will introduce a11y nutrition labels, where developers can document a11y features that the app supports. For auditors it means that they can test mentioned a11y features

<pauljadam> We may need a note about authors putting in the effort to have good readability and layout at the maximum text sizes, like if there is only 1 or 2 letters and lots of text hyphenation - showing then it's not very readable and they need to reflow better, e.g., reflow into 1 column rather than 2 columns. So you can pass WCAG with very hard to read

<pauljadam> large size text as long as it's not truncated but there should be an effort to make it visually easy to read at max text sizes.

<JJ> Jamie: Upcoming apple features: https://www.apple.com/newsroom/2025/05/apple-unveils-powerful-accessibility-features-coming-later-this-year/

Jamie: https://www.apple.com/newsroom/2025/05/apple-unveils-powerful-accessibility-features-coming-later-this-year/

<JJ> Low Vision TF: https://w3c.github.io/low-vision-a11y-tf/requirements.html#text-size

ACTION: Define "assistive technology"

Jamie: in mobile context we can considered a minimum size, which can be AAA requirement to introduce. Also, some text will not be able to reach 200% due to limitations due to the smaller viewport of the mobile screen

ACTION: Define "loss of content or functionality"

ACTION: Note or section about user preferences, including go beyond WCAG requirements?

ACTION: Add note about non-linear scaling (applies on Android and iOS)

ACTION: Provide guidance about determining the text size in percentages

ACTION: Mention large content viewer and other alternatives when content is limited to certain bounds

<Jamie> can we vote today on keeping or not keeping WCAG2ICT notes?

<JJ> Poll: Does 1.4.4 apply directly as written, given that assistive technology is clearly defined?

<Jamie> 0

<Tim> 0

0

<pauljadam> +1 applies as written and assistive technology are defined clearly to include screen magnifiers

<RobW> Can you share a link for the wcag2ict guidance?

<GleidsonRamos> 0

<Karla> 0

<Illai> +1

<JJ> WCAG2ICT guidance: https://www.w3.org/TR/wcag2ict-22/#applying-sc-1-4-4-resize-text-to-non-web-documents-and-software

<Tim> I agree to the statement that assistive technology are defined clearly to include screen magnifiers

<RobW> +1

JJ: WCAG2ICT made a change to the definition of AT, but the normative text is applied as written. But in fact it changes the normative text, because the definition is different

<JJ> Poll: Do we want to include Note 1 of WCAG2ICT as written?

<JJ> The Intent section refers to the ability to allow users to enlarge the text on screen at least up to 200% without needing to use assistive technologies. This means that the application provides some means for enlarging the text 200% (zoom or otherwise) without loss of content or functionality, or that the application works with the platform

<JJ> accessibility features to meet this success criterion.

<Jamie> +1

<Illai> +1

<Tim> +1

<GleidsonRamos> +1

<Karla> +1

<pauljadam> -1 because the text "(zoom or otherwise)" is vague and confusing

<Tanya> -1 for the same reason as pauljadam mentioned

<Jamie> "actually +1 with modification "up to 200%"

<RobW> +1 (but I do have a minor issue as above)

<Jamie> take out the "at least"

<JJ> Poll: Do we want to include Note 2 of WCAG2ICT as written?

<JJ> For non-web software, there may be cases where the platform does not scale all text up to 200%. In such cases, authors are encouraged to meet user needs by scaling text to the extent supported by user settings in the platform.

<pauljadam> -1 does not seem necessary, I'd rather a note about supporting maximum text sizes

<RobW> +1

<Tanya> +1

<Tim> +1

<Karla> +1

<pauljadam> Or maybe this note needs to talk about large content viewer

<Jamie> 0; some note about scaling is needed, with definition of non-linear scaling up to 200%. Not necessarily larger than 200% for this SC

<Illai> +1

<JJ> Summary of poll results for Success Criterion 1.4.4 guidance:

<JJ> Poll: Does 1.4.4 apply directly as written, given that assistive technology is clearly defined?

<JJ> Results: 2x +1 and 5x 0 due to definition clarification

<JJ> Poll: Do we want to include Note 1 of WCAG2ICT as written?

<JJ> Results: 6x +1 and 2x -1 for mentioning "zoom"

<JJ> Poll: Do we want to include Note 2 of WCAG2ICT as written?

<JJ> Results: 5x +1, 1x 0 and 1x -1 for limited scope

JJ: summarized the poll results incl consideration
~~~
</details>

**Summary of poll results for Success Criterion 1.4.4 guidance**

Poll: Does 1.4.4 apply directly as written, given that assistive technology is clearly defined?
- Results: 2x +1 and 5x 0 due to definition clarification

Poll: Do we want to include Note 1 of WCAG2ICT as written?
- Results: 6x +1 and 2x -1 for mentioning "zoom"

Poll: Do we want to include Note 2 of WCAG2ICT as written?
- Results: 5x +1, 1x 0 and 1x -1 for limited scope

**Actions**

- [Define "assistive technology"](https://www.w3.org/2025/05/21-matf-minutes.html#ce77)
- [Define "loss of content or functionality"](https://www.w3.org/2025/05/21-matf-minutes.html#ec2d)
- [Note or section about user preferences, including go beyond WCAG requirements?](https://www.w3.org/2025/05/21-matf-minutes.html#279d)
- [Add note about non-linear scaling (applies on Android and iOS)](https://www.w3.org/2025/05/21-matf-minutes.html#090e)
- [Provide guidance about determining the text size in percentages](https://www.w3.org/2025/05/21-matf-minutes.html#d710)
- [Mention large content viewer and other alternatives when content is limited to certain bounds](https://www.w3.org/2025/05/21-matf-minutes.html#faed)
